README.MD

```diff
@@ -25,14 +25,14 @@ EKS cluster:
  - TBC
 
 Terraform modules for each of the layer could be applied by running the set of command below from the corresponding module directory:
-# TERRAFORM INIT
+### TERRAFORM INIT
 CLUSTER_NAME=<CLUSTER_NAME_HERE> TF_DATA_DIR=/tmp/.terraform_${CLUSTER_NAME}_$(basename "$PWD") bash -c  'terraform init -backend-config="bucket=${CLUSTER_NAME}-tfstate"'
```

Comment:
Can you provide mini documentation about usage of the hiera exporter? For example, I want to run robot tests or deploy legion infra locally. How can I generate a `profile.json` file?